frontend/src/utils/urlutils.tsx

```diff
@@ -93,6 +96,34 @@ export const ADDED_MOBILE_PAGE_URL_TO_NAMES: Record<string, string> = {
   '/': 'Home',
 }
 
+export const NAVIGATION_STRUCTURE = {
+  about: {
+    label: 'About',
+    pages: {
+      [WHAT_IS_HEALTH_EQUITY_PAGE_LINK]: 'What is Health Equity?',
+      [GUN_VIOLENCE_POLICY]: 'Policy Context',
```

Comment:
Should we comment the policy context option out for now until the page is up?

Reply:
sure!